documentation/docs/01-routing.md

```diff
@@ -241,6 +241,25 @@ export async function post({ request }) {
 }
 ```
 
+#### URL parsing
+
+The `url` object is an instance of the standard [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) class at the `searchParams` key. As such, you can access the params sent using query params.
+
+```js
+// @filename: index.js
```

Comment:
typechecking will fail without this

```suggestion
// @filename: ambient.d.ts
declare global {
  const search: (opts: { limit: string | null, offset: string | null }) => Response;
}

export {};

// @filename: index.js
// ---cut---
```

Reply:
Nice!!! I made some confusion because I'm using this with Svelte 😅